settings.gradle.kts

```diff
@@ -45,15 +45,44 @@ dependencyResolutionManagement {
 rootProject.name = "Readium"
 
 include(":readium:adapters:pdfium:pdfium-document")
+project(":readium:adapters:pdfium:pdfium-document")
```

Comment:
By default Gradle will use the folder name as artifact ID, which is not suitable because we have several folders named "navigator", for example. "shared" might also conflit with other modules in a host project.